### PR TITLE
Fixes 1455: allow intropsect of more than one repo

### PR DESCRIPTION
**Pull request by jlsherrill**
and add rhel 8.7 baseos to init containers

## Testing steps
* tests pass
* ` go run cmd/external-repos/main.go import`
*  `go run cmd/external-repos/main.go introspect  https://cdn.redhat.com/content/dist/rhel8/8.7/x86_64/baseos/os https://cdn.redhat.com/content/dist/rhel8/8.6/x86_64/baseos/os`
*  `go run cmd/external-repos/main.go introspect  https://cdn.redhat.com/content/dist/rhel8/8.7/x86_64/baseos/os https://cdn.redhat.com/content/dist/rhel8/8.6/x86_64/baseos/os --force`

### CODE REVIEW

File: cmd/external-repos/main.go [hunk 1]
```diff
@@ -48,33 +48,34 @@ func main() {
...
+			log.Error().Msg("Usage:  ./external_repos introspect [--force] URL [URL2]...")
 			os.Exit(1)
 		}
-		url := args[2]
-		if len(args) > 3 {
-			flagset := flag.NewFlagSet("introspect", flag.ExitOnError)
-			flagset.BoolVar(&forceIntrospect, "force", false, "Force introspection even if not needed")
-			if err = flagset.Parse(args[3:]); err != nil {
-				log.Error().Err(err).Msg("Usage:  ./external_repos introspect URL [--force]")
-				os.Exit(1)
```
Comment (jlsherrill) [inline cmd/external-repos/main.go:60]: note that we should probably switch to a better arg parsing library overall, but i didn't want to do it as part of this change

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMS-1455

**Comment by Andrewgdewar** [comment]
I tested this with some mixed results, will confer with you before acking.

**Review by Andrewgdewar — APPROVED** [review]
Tested with a normal repo, working well

**Pull request merged**